VideoLocker/src/main/java/org/edx/mobile/view/CourseDashboardFragment.java

```diff
@@ -70,6 +70,9 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
             parent = (LinearLayout) view.findViewById(R.id.dashboard_detail);
             shareButton = (ImageButton) view.findViewById(R.id.course_detail_share); //invisible by default
 
+            // Full course name should appear on the course's dashboard screen.
+            courseTextName.setEllipsize(null);
+            courseTextName.setSingleLine(false);
```

Comment:
It's unfortunate that these layout details need to be declared in the code, but I suppose it's the best solution. Another option would have been to change these properties by using a different theme, but I suppose that would have been overkill.